### PR TITLE
blah

### PR DESCRIPTION
**Pull request by vadimberezniker**
---

**Version bump**: TODO

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,8 @@ BuildBuddy is an open source Bazel build event viewer, result store, and remote
 
 It's written in Golang and React and can be deployed as a Docker image. It's run both as a [cloud hosted service](https://buildbuddy.io) and can be deployed to your cloud provider or run on-prem. BuildBuddy's core is open sourced in this repo under the [MIT License](https://github.com/buildbuddy-io/buildbuddy/blob/master/LICENSE).
 
+Hello world. make change.
```
Comment (tylerwilliams) [inline README.md:15]: Hey vadim this looks good but can we say "hello everyone" instead?

File: README.md [hunk 2]
```diff
@@ -12,6 +12,8 @@ BuildBuddy is an open source Bazel build event viewer, result store, and remote
...
+
```
Comment (tylerwilliams) [inline README.md:16]: Remove this newline please

**Review by tylerwilliams — APPROVED** [review]
LGTM

THis looks great, 2 nits, and we can submit!

File: README.md (hunk 1 above)
Comment (vadimberezniker) [reply README.md]: Done.

File: README.md (hunk 2 above)
Comment (vadimberezniker) [reply README.md:16]: Are you sure that's a good idea?

**Comment by siggisim** [comment]
great change!

**Comment by vadimberezniker** [comment]
> great change!

I'll create a launch ticket. Maybe we can ship this somewhere in Q4.

**Comment by tylerwilliams** [comment]
go/launchcal

**Comment by siggisim** [comment]
> go/launchcal

Link wasn't working, but I fixed it.

**Pull request closed without merging**